.travis.yml

```diff
@@ -1,19 +1,10 @@
 sudo: required
-dist: trusty
-language: node_js
-cache:
-  directories:
-    - node_modules
-os:
-  - linux
-  - osx
 
-# This is needed to make NVM happy on OS X https://github.com/travis-ci/travis-ci/issues/6476
-osx_image: xcode7.3
+language: node_js
 
 node_js:
-  - '6'
-  - '8'
+  - 8
```

Comment:
You could use  

``` 
 - "node"
  - "lts/*"
```

This tests latest version of node and the last LTS version.

Reply:
I've switched the `engines` field in `package.json` to Node v8 as the minimum so I'd rather test explicitly against that until we change the minimum again. But testing against both v8 as well as the latest release sounds fine.

(I might just do it for the `linux` build job, though, because OS X is slow...)